### DISCUSSION TITLE
Rename or delete a saved query?

### DISCUSSION THREAD

**Discussion by mtbayley** [datalust/seq-tickets#1946] [Help] [answered]
I can't find how to rename or delete a saved query.

I would expect to have the same options in the editor as when I edit a signal, but this is all I see:

![image](https://github.com/datalust/seq-tickets/assets/34929316/63c57538-ef39-4379-ac3b-e07f364a4194)

**Comment by mtbayley** [comment]
This is version 2023.4.100067

**Comment by KodrAus** [comment, marked as answer]
Thanks for the report @mtbayley! This looks like a bug; I've logged https://github.com/datalust/seq-tickets/issues/1947 to track it in the patch we're currently working on.
